common/utils/src/main/resources/error/error-classes.json

```diff
@@ -2039,6 +2039,12 @@
     },
     "sqlState" : "42K07"
   },
+  "INVALID_SCHEMA_OR_RELATION_NAME" : {
+    "message" : [
+      "`<name>` is not a valid name for tables/schemas. Valid names only contain alphabet characters, numbers and _."
```

Comment:
Please, remove backticks around <name>. We use `toSQLId()`.

Reply:
+1

Reply:
Oops didn't see that when I flipped it.